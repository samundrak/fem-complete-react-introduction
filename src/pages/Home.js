import React from 'react';
import { Row, Col } from 'react-bootstrap';
import VideoCard from '../components/VideoCard';

const Home = () => (
  <Row className="show-grid">
    <Col xs={6} md={3}>
      <VideoCard />
    </Col>
    <Col xs={6} md={3}>
      <VideoCard />
    </Col>
    <Col xs={6} md={3}>
      <VideoCard />
    </Col>
    <Col xs={6} md={3}>
      <VideoCard />
    </Col>
  </Row>
);

export default Home;
