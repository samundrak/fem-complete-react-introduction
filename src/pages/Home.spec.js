import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Home from './Home';

test('Home snapshot test', () => {
  const component = shallow(<Home />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});