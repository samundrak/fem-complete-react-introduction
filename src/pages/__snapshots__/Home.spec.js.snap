// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot test 1`] = `
<Row
  bsClass="row"
  className="show-grid"
  componentClass="div"
>
  <Col
    bsClass="col"
    componentClass="div"
    md={3}
    xs={6}
  >
    <VideoCard />
  </Col>
  <Col
    bsClass="col"
    componentClass="div"
    md={3}
    xs={6}
  >
    <VideoCard />
  </Col>
  <Col
    bsClass="col"
    componentClass="div"
    md={3}
    xs={6}
  >
    <VideoCard />
  </Col>
  <Col
    bsClass="col"
    componentClass="div"
    md={3}
    xs={6}
  >
    <VideoCard />
  </Col>
</Row>
`;
